### PR TITLE
localisation missing on judge and courtstaff

### PR DESCRIPTION
**Pull request by utpalBeehyv**
https://github.com/pucardotorg/dristi/issues/2420
https://github.com/pucardotorg/dristi/issues/2421

## Release Notes

- **New Features**
  - Enhanced internationalization support across various components, including dynamic translations for labels and buttons.
  - Improved search functionality with updated configurations for dropdowns and sorting capabilities in multiple search forms.

- **Bug Fixes**
  - Refined error handling in task detail parsing and improved consistency in data structures.

- **Documentation**
  - Standardized naming conventions for UI components and search fields to enhance clarity and maintainability.

- **Chores**
  - Updated configuration files to reflect new properties and improve data handling across the application.

### CODE REVIEW

**Review by vaibhavct — APPROVED** [review]

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/components/TaskComponentCalander.js [hunk 1]
```diff
@@ -200,7 +200,7 @@ const TaskComponentCalander = ({ isLitigant, uuid, filingNumber, inCase = false
           onClick={handleDownload}
         >
           <DownloadIcon />
-          <div style={{ fontWeight: 700, fontSize: "16px", marginLeft: "10px" }}>Download Cause List</div>
+          <div style={{ fontWeight: 700, fontSize: "16px", marginLeft: "10px" }}>{t("DOWNLAOD_CAUSE_LIST")}</div>
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/hearings/src/components/TaskComponentCalander.js:203]: Done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ export const SummonsTabsConfig = {
   showTab: true,
   SummonsTabsConfig: [
     {
-      label: "Pending",
+      label: "PENDING",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js:13]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js [hunk 3]
```diff
@@ -119,31 +119,31 @@
...
               },
               {
-                label: "Delivery Channel",
+                label: "DELIEVERY_CHANNEL",
                 jsonPath: "taskDetails",
                 additionalCustomization: true,
               },
               {
-                label: "Issued",
+                label: "ISSUED",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js:146]: No Need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js [hunk 4]
```diff
@@ -321,10 +322,11 @@ export const TabFSOSearchConfig = {
                 disable: false,
                 populators: {
                   name: "substage",
+                  optionsKey: "code",
                   mdmsConfig: {
                     masterName: "SubStage",
                     moduleName: "case",
-                    select: "(data) => {return data['case'].SubStage?.map((item) => {return item.subStage;});}",
+                    select: "(data) => {return data['case'].SubStage?.map((item) => {return item});}",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js:329]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js [hunk 5]
```diff
@@ -119,31 +119,31 @@
...
               },
               {
-                label: "Order Type",
+                label: "ORDER_TYPE",
                 jsonPath: "taskType",
                 additionalCustomization: true,
               },
               {
-                label: "Delivery Channel",
+                label: "DELIEVERY_CHANNEL",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/configs/SuumonsConfig.js:141]: irrelavant

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js [hunk 6]
```diff
@@ -174,10 +174,11 @@
...
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/FSOHomeConfig.js:181]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/BenchHomeConfig.js [hunk 7]
```diff
@@ -295,10 +295,11 @@ export const TabBenchSearchConfig = {
                 disable: false,
                 populators: {
                   name: "substage",
+                  optionsKey: "code",
                   mdmsConfig: {
                     masterName: "SubStage",
                     moduleName: "case",
-                    select: "(data) => {return data['case'].SubStage?.map((item) => {return item.subStage;});}",
+                    select: "(data) => {return data['case'].SubStage?.map((item) => {return item});}",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/BenchHomeConfig.js:302]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/pages/employee/ReviewSummonsNoticeAndWarrant.js [hunk 8]
```diff
@@ -273,8 +273,8 @@ const ReviewSummonsNoticeAndWarrant = () => {
     if (rowData?.taskDetails || nextHearingDate) {
       const caseDetails = handleTaskDetails(rowData?.taskDetails);
       return [
-        { key: "Issued to", value: caseDetails?.respondentDetails?.name },
-        { key: "Issued Date", value: convertToDateInputFormat(rowData?.createdDate) },
+        { key: "ISSUE_TO", value: caseDetails?.respondentDetails?.name },
+        { key: "ISSUE_DATE", value: convertToDateInputFormat(rowData?.createdDate) },
         // { key: "Next Hearing Date", value: nextHearingDate?.startTime ? formatDate(nextHearingDate?.startTime) : "N/A" },
         { key: "AMOUNT_PAID_TEXT", value: `Rs. ${caseDetails?.deliveryChannels?.fees || 100}` },
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/orders/src/pages/employee/ReviewSummonsNoticeAndWarrant.js:279]: no need

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js [hunk 9]
```diff
@@ -468,15 +483,15 @@
...
+        case "STATUS":
           return t(value); // document status
-        case "Issued":
+        case "ISSUED":
           return `${formatDateDifference(value)} days ago`;
-        case "Order Type":
+        case "ORDER_TYPE":
           return t(value);
-        case "Delivery Channel":
+        case "DELIEVERY_CHANNEL":
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/UICustomizations.js:494]: irrelavnt

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/HomeConfig.js [hunk 10]
```diff
@@ -199,14 +199,14 @@ export const rolesToConfigMapping = [
...
-  { code: "application-order-submission-feedback", name: "Order Submission Feedback" },
-  { code: "application-order-submission-default", name: "Order Submission" },
-  { code: "artifact-default", name: "Evidence" },
-  { code: "task-summons", name: "Task Summons" },
+  { code: "case-default", name: "WBH_MDMS_MASTER_CASE" },
+  { code: "hearing-default", name: "CS_HEARING" },
+  { code: "order-default", name: "CS_ORDER" },
+  { code: "application-voluntary-submission", name: "VOLUNTARY_SUBMISSION" },
+  { code: "application-order-submission-feedback", name: "ORDER_SUBMISSION_FEEDBACK" },
+  { code: "application-order-submission-default", name: "ORDER_SUBMISSSION" },
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/home/src/configs/HomeConfig.js:207]: done

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCasesConfig.js [hunk 11]
```diff
@@ -470,7 +476,7 @@ export const TabSearchconfig = {
                     moduleName: "Evidence",
                     localePrefix: "EVIDENCE_TYPE",
                     select:
-                      "(data) => {return data['Evidence'].EvidenceType?.map((item) => {return { ...item, name: item.subtype && item.subtype !== '' ? `${item.type} (${item.subtype})` : item.type };});}",
+                      "(data) => {return data['Evidence'].EvidenceType?.map((item) => {return { ...item, name: item.subtype && item.subtype !== '' ? `${item.type}_${item.subtype}` : item.type };});}",
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/pages/employee/AdmittedCases/AdmittedCasesConfig.js:479]: no required

File: frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js [hunk 12]
```diff
@@ -686,22 +686,22 @@ export const UICustomizations = {
...
+        case "DOCUMENT_TYPE":
           return <Evidence userRoles={userRoles} rowData={row} colData={column} t={t} value={value} showAsHeading={true} />;
-        case "Hearing Type":
-        case "Source":
-        case "Status":
+        case "HEARING_TYPE":
+        case "SOURCE":
+        case "STATUS":
           //Need to change the shade as per the value
           return <CustomChip text={t(value)} shade={value === "PUBLISHED" ? "green" : "orange"} />;
```
[the remark this answers is not in the document]
Comment (utpalBeehyv) [reply frontend/micro-ui/web/micro-ui-internals/packages/modules/dristi/src/configs/UICustomizations.js:699]: no need

**Pull request merged**